tutorials/single_phase_flow.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I think "final solution" is better than "end solution".</p><p>Here, I would only reference the latter tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>